Makefile

```diff
@@ -180,7 +180,7 @@ apply-xgboost-cpu-local: libxgboost-cpu-local pipxgboost
 
 pipxgboost:
 	@echo "----- pip install xgboost built locally -----"
-	cd xgboost/python-package/dist && pip install xgboost-0.7-py3-none-any.whl --upgrade --target ../
+	cd xgboost/python-package/dist && pip install *.whl --upgrade --target ../
```

Comment:
Will there only be 1 .whl file?

Reply:
Ya, the directory is cloned freshly.  We can see if jenkins has issues, seems not so far.